controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/BillingMaintainer.java

```diff
@@ -28,10 +28,9 @@ public void maintain() {
                 .filter(tenant -> tenant instanceof CloudTenant)
                 .map(tenant -> (CloudTenant) tenant)
                 .forEach(cloudTenant -> controller().applications().asList(cloudTenant.name())
```

Comment:
Return value here changed from all instances to all applications — I believe the former would have multiple billing if anyone had multiple instances, as it looks to me like the billing info is gathered from anything matching tenant + application. The current avoids this. It's also possible — probable even — that multiple billing is also guarded against somewhere else. 
PTAL @olaaun.

Reply:
LGTM